### DISCUSSION TITLE
Changing root folder letter to '/' in a custom PowerShell provider

### DISCUSSION THREAD

**Discussion by yotsuda** [PowerShell/PowerShell#20157] [Enhancements]
Hello experts,
I'm attempting to create my own provider class in PowerShell. In this custom provider, I want the root folder to be represented by a / instead of the default \\.

I've subclassed NavigationCmdletProvider and overridden ItemSeparator to return /. However, the representation of the root folder still remains as \\.

How can I ensure that the root folder uses the / character in my custom provider?
Thank you in advance for any guidance.

**Comment by yotsuda** [comment]
By specifying parameters in the PSDriveInfo constructor, I was able to change the internal root folder character to /.
However, the character of the root folder displayed on the PS console remains \\.

How can I fix it? Thanks.

**Reply by oising** [reply]
Is this on Windows or Linux? If A, have you tried B and vice-versa?

**Reply by yotsuda** [reply]
Hello @oising ,

Thank you for your reply. This is on Windows. I've decided to use backslashes for my provider instead of slashes. It works well now. Thank you.
